README.md

```diff
@@ -1,34 +1,44 @@
-# GRASS Addons git repository
+# GRASS GIS Addons
 
-## How to get write access here
+This is the official GRASS GIS Addons git repository.
+Head over to [GRASS GIS 7 Addons Manual
+pages](https://grass.osgeo.org/grass78/manuals/addons/)
+for an overview.
 
-Read access is granted to the public, write access
-must be requested, see here for details:
-<https://trac.osgeo.org/grass/wiki/HowToContribute#WriteaccesstotheGRASSaddonsrepository>
+## Install AddOns
 
-However, it is also possible to fork this repository, insert your AddOn or fix
-an existing one in a new branch and finally open
-a [pull request](https://help.github.com/en/articles/about-pull-requests).
+The simplest way to install GRASS GIS Addons is to use the `g.extension`
+module which is part of any GRASS GIS installation:
 
-In either case, please read the submitting rules at the bottom of this page.
+<https://grass.osgeo.org/grass-stable/manuals/g.extension.html>
+
+The same module can also remove installed addons.
+
+## This repository
 
-## How to get the AddOn code
+### Clone locally
 
-Clone of the entire AddOns git repository:
+Clone of the entire addons git repository:
 
 ```
 git clone https://github.com/OSGeo/grass-addons.git grass_addons
 ```
 
-## How to install or remove AddOns in your GRASS installation
+### Write access
```

Comment:
What about changing this to **Contributing**? Then, adding a paragraph like (compiled/adapted according to comments):

Contributing to GRASS GIS addons repository is really simple. Just fork the repo and make a [pull request](https://help.github.com/en/articles/about-pull-requests). We welcome fixes, enhancements and new addons.
If you would like to become a mantainer, you can request for write access. See [here](https://trac.osgeo.org/grass/wiki/HowToContribute#WriteaccesstotheGRASSaddonsrepository) for details.
In either case, please read the submitting rules at the bottom of this page.